src/Room/ChatRoomAclDataAccessor.php

```diff
@@ -5,7 +5,10 @@
 use Amp\Artax\HttpClient;
 use Amp\Artax\Response as HttpResponse;
 use Amp\Promise;
+use function Amp\resolve;
```

Comment:
Use fqn at call time rather than import, mainly because `resolve()` exists in amp/dns as well

Reply:
Ah, makes sense!